### PR TITLE
One-line install, no repo change.

### PR DESCRIPTION
**Pull request by lordbagel42**

### CODE REVIEW

**Comment by lordbagel42** [comment]
nope wget screams and yells like a toddler if you don't

**Comment by ButterscotchV** [comment]
I think I just did `pkg update` after and it worked? idk, if this works without that then that's probably fine too

File: tools/termux-installation.md [hunk 1]
```diff
@@ -35,14 +33,11 @@ apt install openssl wget && wget https://raw.githubusercontent.com/lordbagel42/s
...
-4. Open Termux, and run the following commands.
+4. Open Termux, and run the following command:
 
 ```
-apt update
-termux-change-repo # SELECT MAIN THEN BFSU.
-apt install openssl wget && wget https://raw.githubusercontent.com/lordbagel42/slimevr-termux-installer/main/install.sh && chmod +x install.sh && ./install.sh
+pkg install openssl wget && wget https://raw.githubusercontent.com/lordbagel42/slimevr-termux-installer/main/install.sh && chmod +x install.sh && ./install.sh
 ```
-
```
Comment (Louka3000) [inline tools/termux-installation.md:45]: Why remove that newline? Not sure if it changes anything tho.

Comment (lordbagel42) [reply tools/termux-installation.md:45]: I didn't even notice I removed it lol.

Comment (Louka3000) [reply tools/termux-installation.md:45]: Can you add it back (even if it doesn’t change anything, it looks better in the code :p)

**Review by Louka3000 — APPROVED** [review]

**Review by calliePepper — APPROVED** [review]
Nice changes

**Pull request merged**